src/item_factory.cpp

```diff
@@ -2886,7 +2882,7 @@ void Item_factory::load_basic_info( const JsonObject &jo, itype &def, const std:
     assign( jo, "explode_in_fire", def.explode_in_fire );
     assign( jo, "insulation", def.insulation_factor );
     assign( jo, "solar_efficiency", def.solar_efficiency );
-    assign( jo, "ascii_picture", def.picture_id );
+    assign( jo, "id", def.picture_id );
```

Comment:
this is wildly incorrect. 
a) it should be possible to specify a different picture (in the case where multiple things might have the same ascii art, which is very plausible)
b) if you really need to use id to initialize this, you can just use the existing id that's already been read in, instead of trying to read it in again